changelog.d/10343.bugfix

```diff
@@ -0,0 +1 @@
+Redaction events were previously not properly purged as part of the purge history job. Leaving behind some (unexpected) redaction events when backfilling. Redaction events will now be purged and if (already) left behind ignored.  Contributed by Andreas Rammhold (@andir).
```

Comment:
please could you shorten this a bit? "Fix errors during backfill caused by previously purged redaction events"? See the existing entries at https://github.com/matrix-org/synapse/blob/develop/CHANGES.md for other examples.

Reply:
Done